stdlib/public/stubs/FoundationHelpers.mm

```diff
@@ -33,26 +33,22 @@
 
 static CFHashCode(*_CFStringHashCString)(const uint8_t *bytes, CFIndex len);
 static CFHashCode(*_CFStringHashNSString)(id str);
-static CFTypeID(*_CFGetTypeID)(CFTypeRef obj);
-static CFTypeID _CFStringTypeID = 0;
+static id(*_CFStringCreateTaggedPointerString)(const uint8_t *bytes, CFIndex numBytes);
+static __swift_uint8_t(*_NSIsNSString)(id arg);
 static swift_once_t initializeBridgingFuncsOnce;
 
 extern "C" bool _dyld_is_objc_constant(DyldObjCConstantKind kind,
                                        const void *addr) SWIFT_RUNTIME_WEAK_IMPORT;
 
 static void _initializeBridgingFunctionsImpl(void *ctxt) {
-  auto getStringTypeID =
-    (CFTypeID(*)(void))
-    dlsym(RTLD_DEFAULT, "CFStringGetTypeID");
-  assert(getStringTypeID);
-  _CFStringTypeID = getStringTypeID();
-  
-  _CFGetTypeID = (CFTypeID(*)(CFTypeRef obj))dlsym(RTLD_DEFAULT, "CFGetTypeID");
-  _CFStringHashNSString = (CFHashCode(*)(id))dlsym(RTLD_DEFAULT,
-                                                   "CFStringHashNSString");
-  _CFStringHashCString = (CFHashCode(*)(const uint8_t *, CFIndex))dlsym(
-                                                   RTLD_DEFAULT,
-                                                   "CFStringHashCString");
+  void *cf = dlopen("/System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation", RTLD_LAZY | RTLD_NOLOAD | RTLD_LOCAL | RTLD_FIRST);
```

Comment:
I think hardcoding this path is what we want, but I'm open to feedback to the contrary

Reply:
If it'll run on embedded too, then you'll need "/System/Library/Frameworks/CoreFoundation.framework/CoreFoundation".  But that should also work on macOS

Reply:
ah cool, will change